src/ert/parsing/config_errors.py

```diff
@@ -10,63 +12,68 @@ class ConfigWarning(UserWarning):
 class ConfigValidationError(ValueError):
     def __init__(
         self,
-        errors: Union[str, List[Tuple[Optional[str], str]]],
+        errors: Union[str, List[Union[ErrorInfo, str, Tuple[Optional[str], str]]]],
```

Comment:
Is there still a reason to have the list contain `Tuple[Optional[str], str]` ? I think its only config_parser.py that uses that now, so it is probably less complicated to just convert it to use `ErrorInfo`.

Reply:
Fixed